modules/index/src/test/java/org/apache/ignite/internal/index/IndexBuildControllerTest.java

```diff
@@ -301,8 +302,8 @@ private int indexId(String indexName) {
         return getIndexIdStrict(catalogManager, indexName, clock.nowLong());
     }
 
-    private TablePartitionId replicaId(int partitionId) {
-        return new TablePartitionId(tableId(), partitionId);
+    private ZonePartitionId replicaId(int partitionId) {
+        return new ZonePartitionId(0, partitionId, tableId());
```

Comment:
Same as above, not valid to use 0 here as zoneId, please retrieve the identifier from catalog descriptor like `tableId()` does.